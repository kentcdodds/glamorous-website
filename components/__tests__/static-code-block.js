import React from 'react'
import {mount} from 'enzyme'
import toJson from 'enzyme-to-json'
import staticCodeBlock from '../static-code-block'

test('renders HTML code block string with the pragma language classname', () => {
  const elements = staticCodeBlock({
    code: '<span>// some code</span>',
    language: 'javascript',
  })
  const wrapper = mount(<div>{elements}</div>)
  expect(toJson(wrapper)).toMatchSnapshot()
})
