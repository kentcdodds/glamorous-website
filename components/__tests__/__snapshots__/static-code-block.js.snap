// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders HTML code block string with the pragma language classname 1`] = `
<div>
  <CodeBlock
    code="<span>// some code</span>"
    language="javascript"
  >
    <pre
      className="language-javascript"
    >
      <code
        className="language-javascript"
        dangerouslySetInnerHTML={
          Object {
            "__html": "<span>// some code</span>",
          }
        }
      />
    </pre>
  </CodeBlock>
</div>
`;
